### ISSUE TITLE
Templates Have Changed Since Videos

### ISSUE THREAD

**Issue by invictus2010** [hackingthemarkets/alpaca-react-native#2]
Hey there!

**Expected:** Select the template with tabs after running `expo init` `Navigation` folder shall contain `MainTabNavigator.js`

**Actual:** Template comes with `BottomTabNavigator.js` that contains code different than one contained in video. 

Wasn't able to follow along step-by-step in the video because of the issue.

**Comment by davisdre** [comment]
Yeah, from the video to now it went from JavaScript base to TypeScript based in the template, and I've come to learn that the two have some differences.
